msteams-platform/concepts/build-and-test/share-to-teams-from-web-apps.md

```diff
@@ -152,6 +274,17 @@ You can select to set default values for the following fields on the Share to Te
 ></span>
 ```
 
+The following are the launcher.js definitions:
```

Comment:
We have two sections of launcher.js definition in this doc which might be confusing for developers. we should consolidate them.